### ISSUE TITLE
Changing the value of the Backlight widget sets it to 100%

### ISSUE THREAD

**Issue by qwenger** [qtile/qtile#1868]
# Issue description

When changing the backlight value e.g. by scrolling on the widget the value gets set to 100% no matter what.

I think that it is caused by https://github.com/qtile/qtile/commit/b524e4427ca43f77d022aa05d85dc16a1aef7215#diff-49f3d455c9019a091f0f74c63fb8aff6: `xbacklight` expects a percentage, not the actual value found in the ACPI file (see https://linux.die.net/man/1/xbacklight). That commit removed the conversion to percentages.

For reference, on my computer:
```
$ cat /sys/class/backlight/intel_backlight/max_brightness
24242
```

# Qtile version

Version 0.16.1 (current from https://www.archlinux.org/packages/community/x86_64/qtile/). Used to work but I'm not sure of which version.

**Comment by m-col** [comment]
Can you post your config for the widget?

I think it might be that [this line](https://github.com/qtile/qtile/blob/b524e4427ca43f77d022aa05d85dc16a1aef7215/libqtile/widget/backlight.py#L79)
should be executed only if `self.change_command` is None.

**Comment by qwenger** [comment]
Nothing fancy really:

```
screens = [
    Screen(
        bottom=bar.Bar(
            [
                ...
                widget.Backlight(backlight_name="intel_backlight", step=5),
                ...
            ]
        )
    )
]
```

No that line is not problematic, or only as a secondary problem. In `button_press` the absolute value is calculated (the value w.r.t. the maximum value from the ACPI file) and passed to `change_backlight` and from there to the `change_command`. However the default `change_command` uses `xbacklight`, which expects a percentage value rather than absolute.

Please compare with the previous implementation: https://github.com/qtile/qtile/blob/5ea22c655b4591c1603724e28431293306679c5d/libqtile/widget/backlight.py#L105

**Comment by m-col** [comment]
Please could you try #1871 and see if that commit fixes it?

**Comment by qwenger** [comment]
Thank you very much, yes that does the trick!

**Comment by ramnes** [comment]
Fixed in 4d4636efe9a8771e6272c64d90e7e22fbdf4e095
